src/main/kotlin/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,47 @@
+package calculator
+
+class StringCalculator {
+    companion object {
+        private const val DELIMITER = " "
+    }
+
+    fun calculate(stringExpression: String?): Scalar {
+        if (stringExpression == null || stringExpression.isBlank()) {
```

Comment:
`String?`에 대해 호출할 수 있는 `isNullOrBlank()`가 있습니다.

```suggestion
        if (stringExpression.isNullOrBlank()) {
```